DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/styles/Site.css

```diff
@@ -0,0 +1,22 @@
+﻿.glyphicon-lg{font-size:3em}
```

Comment:
hey. The name of file is awful :) you shall call it "stats". not site. and without uppercase